app/vep/vep_resources.py

```diff
@@ -15,21 +15,54 @@
 limitations under the License.
 """
 import logging
+import tempfile 
+import os, json
+  
 from typing import Annotated
-
+from core.config import VEP_CONFIG_INI_PATH
 from fastapi import APIRouter, responses, UploadFile, Form
 
-from core.logging import InterceptHandler
-
-logging.getLogger().handlers = [InterceptHandler()]
+# from core.logging import InterceptHandler
+from models.pipeline_model import ConfigIniParams
+# logging.getLogger().handlers = [InterceptHandler()]
 
 router = APIRouter()
 
-
-
 @router.post("/submissions", name="submit_vep")
 def submit_vep(genome_id:Annotated[str, Form()], input_file: UploadFile):
     try:
         return {"genome_id":genome_id,"filename": input_file.filename}
     except Exception as e:
         logging.info(e)
+
+# Creates config ini file
+def create_config_ini_file(symbol: bool = "true", biotype: bool = "true"):
```

Comment:
Forgive the stupid question; but isn't `"true"` a string (`str`) rather than a bool? And don't we expect the actual booleans (e.g.`True`) as default?

By the way; I think the defaults should probably be false.

Reply:
yes, my bad. I didnt know it was `True` and not `true` which it didnt like.

Reply:
That your editor allowed you to write `pullLatest: bool = "true"` without complaint probably suggests that type checking isn't working in your editor. I wonder if it's worth running type checking automatically upon commit as part of automated tests. @kamaldodiya @Mehrnaz-Charkhchi @veidenberg what do you think?